src/hostapi/pulseaudio/pa_linux_pulseaudio.c

```diff
@@ -750,10 +755,8 @@ PaError PaPulseAudio_Initialize( PaUtilHostApiRepresentation ** hostApi,
     {
         if( pulseaudioHostApi->allocations )
```

Comment:
```suggestion
        if( lockTaken )
```

Reply:
I like this. It is clearer.

add
```
    int lockTaken = 0;
```

to top of function, and

```
    lockTaken = 1;
```

after the call to `PaPulseAudio_Lock`

Reply:
Ok this should be also resolved